### DISCUSSION TITLE
Come fare una segnalazione di errore

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#576] [Annunci]
Per segnalare un errore dell'applicazione è necessario fornire il maggiorn numero di informazioni possibile: indicare semplicemente che qualcosa _"non funziona"_ non è di nessuna utilità.

Se si vuole risolvere il problema si dovrà:
- anzi tutto, assicurarsi di aver aggiornato il registro all'ultima versione rilasciata;
- [aprire una segnalazione di errore](https://github.com/iisgiua/giuaschool/issues/new?assignees=&labels=Errore&template=bug-report.md&title=) (una segnalazione distinta per ciascun problema);
- descrivere quale è il problema incontrato e quando si verifica;
- riportare in allegato al messaggio il _log degli errori_.

Per generare il _log degli errori_, seguire la procedura:
- come utente _amministratore_, andare alla pagina **Sistema > Manutenzione** e abilitare il **debug**;
- eseguire la pagina che causa l'errore e segnarsi l'ora in cui viene eseguita;
- tornare alla pagina **Sistema > Manutenzione** e estrarre il log, indicando l'ora che è stata segnata in precedenza;
- scaricare il file di log e allegarlo al messaggio di segnalazione (**non fare copia e incolla del testo del log, ma allegare il file**)
- tornare alla pagina **Sistema > Manutenzione** e disabilitare il **debug**.

**Comment by Davide031** [comment]
Appena faremo gli scrutini glielo manderò
